scipy/signal/_savitzky_golay.py

```diff
@@ -123,12 +123,7 @@ def savgol_coeffs(window_length, polyorder, deriv=0, delta=1.0, pos=None,
         x = x[::-1]
 
     order = np.arange(polyorder + 1).reshape(-1, 1)
-    if order.size == 1:
-        # Avoid spurious DeprecationWarning in numpy 1.8.0 for
-        # ``[1] ** [[2]]``, see numpy gh-4145.
-        A = np.atleast_2d(x ** order[0, 0])
-    else:
-        A = x ** order
+    A = x ** order
```

Comment:
I believe the fix was backported to 1.8.2 in https://github.com/numpy/numpy/pull/4419, though I could be mistaken. I guess I wasn't 100% sure how hard the distinction between 1.8.2 vs 1.8.0 should be.